.travis.yml

```diff
@@ -33,6 +33,7 @@ script:
   - if [ $TRAVIS_OS_NAME == 'linux' ] && [ $TRAVIS_BRANCH == 'master' ] && [ $TRAVIS_PULL_REQUEST == 'false' ]; then
       docker build -t kbclient .;
     fi
+  - cd ../shared && npm i --global flow-bin@`tail -n1 .flowconfig` && flow
```

Comment:
this is fine, we could theoretically install https://github.com/npm/node-semver then do semver.valid() on it